packages/fgtclb/academic-projects/Classes/Upgrades/FlexFormUpgradeWizard.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FGTCLB\AcademicProjects\Upgrades;
+
+use TYPO3\CMS\Core\Database\Connection;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Install\Attribute\UpgradeWizard;
+use TYPO3\CMS\Install\Updates\DatabaseUpdatedPrerequisite;
+use TYPO3\CMS\Install\Updates\UpgradeWizardInterface;
+
+#[UpgradeWizard('academicProjects_flexFormUpgradeWizard')]
+final class FlexFormUpgradeWizard implements UpgradeWizardInterface
+{
+    private const MIGRATE_CONTENT_TYPES_LIST = [
+        'academicprojects_projectlist',
+        'academicprojects_projectlistsingle',
+    ];
+
+    private const MIGRATE_PLUGIN_SETTINGS = [
+        'setting.filter.options' => 'settings.hideFilter',
```

Comment:
```suggestion
        'settings.filter.options' => 'settings.hideFilter',
```

Typo in the `old option name`. Should be `settings` instead of `setting` - mind the `s`.